### ISSUE TITLE
Multiple custom categories

### ISSUE THREAD

**Issue by Ionys320** [joeattardi/picmo#200]
Hi,
Is it planned to be able to have multiple custom categories ? For several custom characters with their own emojis, for example.

Thanks!

**Comment by joeattardi** [comment]
Yes! This is in the roadmap. I didn't want to delay 5.0.0 any longer than I already had so several things - including multiple custom categories - got pushed. Definitely on the list though.

I will leave this issue open as a reminder! 😄
